qiskit_experiments/curve_analysis/standard_analysis/oscillation.py

```diff
@@ -126,7 +126,7 @@ def _evaluate_quality(self, fit_data: curve.CurveFitResult) -> Union[str, None]:
         return "bad"
 
 
-class DumpedOscillationAnalysis(curve.CurveAnalysis):
```

Comment:
Please use this decorator, rather than simply removing the old class. 
https://github.com/Qiskit/qiskit-experiments/blob/511e33b09d91c12d8a526c3818a703156fe5c8e6/qiskit_experiments/warnings.py#L71-L76

This is because our experiment loading mechanism depends on the path to the class, so changing the class name or module name will immediately invalidate the experiment entries saved in the result database. We need to show deprecation warning for at very least one release cycle to give users a chance of migration.

Reply:
Sure @nkanazawa1989